Dbal/SchemaAssetFilterInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\Dbal;
+
+/**
+ * Classes consumed by SchemaAssetsFilterManager to filter DBAL assets.
+ */
+interface SchemaAssetFilterInterface
```

Comment:
This creates a new API while DBAL works with a callable here.

Why not relying on the DBAL signature directly (and `SchemaAssetsFilterManager` becomes a callable implementing a `All` feature over a list of other callables returning a boolean).